app/helpers/budget_warning.rb

```diff
@@ -13,20 +13,18 @@ def initialize
     end
 
     def go_live?
-      (Date.today >= date) ? true : false
+      Time.zone.today >= date ? true : false
     end
   end
 
   def display_banner_warning?
-    BudgetWarning::Announcement.new.go_live?
+    BudgetWarning::Announcement.new.go_live? && whitelisted?
   end
   module_function :display_banner_warning?
 
   def whitelisted?
-    whitelist = ['/en/tools/redundancy-pay-calculator/',
-                 '/cy/tools/cyfrifiannell-tal-diswyddo/']
+    whitelist = YAML.load_file('config/budget_warning_tools.yml')
```

Comment:
@margOnline should we move from [DEV APPROVED] to [WIP] until the minor _concern_ above is resolved?